flexmeasures/api/v3_0/tests/test_assets_api.py

```diff
@@ -359,3 +359,30 @@ def test_delete_an_asset(client, setup_api_test_data, requesting_user):
     assert delete_asset_response.status_code == 204
     deleted_asset = GenericAsset.query.filter_by(id=existing_asset_id).one_or_none()
     assert deleted_asset is None
+
+
+@pytest.mark.parametrize(
+    "requesting_user",
+    ["test_prosumer_user@seita.nl"],
+    indirect=True,
+)
+def test_get_assets_new(
+    client,
+    setup_api_test_data,
+    setup_accounts,
+    requesting_user,
+):
+    """
+    Get assets in the account the Prosumer account is in the
+    consultancy account of
+    """
+    account_name = "Consultant"
+    query = {"account_id": setup_accounts[account_name].id}
+
+    get_assets_response = client.get(
+        url_for("AssetAPI:index"),
+        query_string=query,
+    )
+    print("Server responded with:\n%s" % get_assets_response.json)
+    assert get_assets_response.status_code == 200
+    assert len(get_assets_response.json) == 1
```

Comment:
There should also be a test showing that the consultant cannot do things other than read, e.g. using the patch endpoint.

Reply:
Added test to show the assets can not be patched by a consultant